plugins/bazaar-backend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@backstage/plugin-bazaar-backend",
+  "version": "0.1.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.9.2",
+    "@backstage/config": "^0.1.5",
+    "@types/express": "^4.17.6",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
+    "knex": "^0.95.10",
```

Comment:
Last thing here, can we re-use the semver that we've got already `scaffolder-backend` etc so we don't pull in duplicates. 

```
    "knex": "^0.95.1",
```